ChefExtensionHandler/bin/chef-install.sh

```diff
@@ -106,7 +123,16 @@ install_from_repo_centos(){
   curl_exit_code=$?
   curl_status $curl_exit_code $yum_repo_config_url $yum_repo_path
 
-  yum -y install chef
+  echo "Installing chef-client package"
+  chef_version=$(get_chef_version)
+  if [ "$chef_version" = "No config file found !!" ]; then
+    echo "Configuration error. Azure chef extension Settings file missing."
```

Comment:
I think default should be latest. If user not specifies version then we should download and install latest instead of existing from here?